devconf-outreachy/source/index.rst

```diff
@@ -0,0 +1,415 @@
+
+* Code reviews
+
+  - Don't take it personally (really don't)
+  - But take responsibility for the code you write
+  - Essential part of programming
+  - Opportunity to learn
+
+* Asking questions
+
+* Working from home/remotely
+
+  - Self-starter
+  - Motivation
+  - Dedicated place to work
+  - Video calls
+
+.. Things to talk about ^ (Aija)
+
+  Code reviews are very demanding and question every single bit.
+
+  "You are not your code" - beware not to take it personally.
+
+  Avoid the opposite extreme, not taking responsibility for the code.
+
+  Important part of software development. Still productive time, even
+  if not actively writing new code.
+
+  Learned and researched new things, some of which did not make it
+  into the submitted contribution.
+
+  Finding a balance when to ask for help -- Is it too soon? Should I
+  try to figure it out myself?
+
+  Working outside the regular office may be challenging. Need to have
+  right motivation and environment. Separate work from leisure and
+  leisure from work.
+  Regular calls for synchronous/instant communication.
+
+Case study: Becoming better together
+------------------------------------
+
+* Industry is influential
+* Silos exist
+* How we came together to everyone's benefit
+
+.. Things to talk about ^ (rpioso)
+
+  Industry is heavily involved in OpenStack and highly influential when it
+  comes to standardization.
+
+  The standards and related documentation are not always ideal, e.g.,
+  standards may be confusing or lacking features. Go figure!
+
+  Finally, young engineers may be hesitant to reach out to the technical
+  "officials".
+
+  So, through the internship, we came together by bringing down the silos
+  between the manufacturers, standards body, and open source community. Here
+  is how our story unfolded:
+
+  * Aija was working on implementing a new and evolving standardized protocol
+    for OpenStack's Ironic project
+  * A couple points of confusion arose
+  * Aija reached out to Dell EMC, the hardware manufacturer implementing the
+    same protocol on the hardware side
+  * Dell engineers connected Aija with the standards body, the Distributed
+    Management Task Force (DMTF) Redfish Forum, to clarify technical matters
+  * Aija's experience has been fed back to the standards organization about
+    what:
+
+    - helped to clarify and improve the technicalities of the standard,
+    - set up the ties between the computer manufacturer's engineers,
+      OpenStack engineers, and standards organization, and
+    - taught Aija to negotiate intricate and complicated technical matters
+      with fellow engineers
+
+  The result proves her internship was successful.
+
+How to be a better mentor
+-------------------------
+
+Work-wise:
+
+* Keep your intern busy, challenged, and growing
+* Help keep them focused to get things done
+* Build on small successes, plan and promote small steps
+
+Community:
+
+* Network interns with the upstream community
```

Comment:
Change "Network interns" to "Help interns network"

Reply:
Done